cmd/set/theme/theme.go

```diff
@@ -0,0 +1,253 @@
+		return nil
+	} else {
+		fmt.Printf("Previous changes will be lost. Press Y to continue: ")
+		fmt.Scanf("%s", &option)
+		if option != "Y" {
+			return nil
+		}
+	}
+
+	fmt.Println("Reseting current theme...")
+	err = resetCalls()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Fetching Auth theme url...")
+	err = themeurl()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Updating CSS based on new theme...")
+	err = hostedPageCalls()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Fetching data...")
+	err = otherCalls()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Updating to new theme...")
+	err = updateCalls()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Your theme has been changed")
+
+	return nil
+}
+
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func getTheme() (string, error) {
+	resp, err := api.GetPage()
+	if err != nil {
+		return "", err
+	}
+	index := resp.Pages[0].Status
+	return cmdutil.ThemeMap[index], nil
+}
+
+func resetCalls() error {
+	conf := config.GetInstance()
+	reset := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/reset"
+	bodyAuth, _ := json.Marshal(map[string]string{
+		"pageType": "Auth",
+	})
+	_, err := request.Rest(http.MethodPut, reset, nil, string(bodyAuth))
+	if err != nil {
+		return err
+	}
+	bodyProfile, _ := json.Marshal(map[string]string{
+		"pageType": "Profile",
+	})
+	_, err = request.Rest(http.MethodPut, reset, nil, string(bodyProfile))
+	if err != nil {
+		return err
+	}
+
+	return nil
+
+}
+
+func updateCalls() error {
+	conf := config.GetInstance()
+	update := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/update"
+	var customJSAuth body
+	var customJSProf body
+	cjs := CustomJS{
+		Content:  "options = {\n    \"language\": \"English\"\n}",
+		fileName: "lr-interface-options",
+	}
+	customJSAuth.CustomJS = append(customJSAuth.CustomJS, cjs)
+	customJSProf.CustomJS = append(customJSProf.CustomJS, cjs)
+
+	customJSAuth.PageType = "Auth"
+	body1, _ := json.Marshal(customJSAuth)
+	_, err := request.Rest(http.MethodPost, update, nil, string(body1))
+	if err != nil {
+		return err
+	}
+	customJSProf.PageType = "Profile"
+	body2, _ := json.Marshal(customJSProf)
+	_, err = request.Rest(http.MethodPost, update, nil, string(body2))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func otherCalls() error {
+	conf := config.GetInstance()
+	profile := conf.AdminConsoleAPIDomain + "/deployment/hostedPage/script/Profile"
+	bodyProfile, _ := json.Marshal(map[string]string{
+		"url": "https://hosted-pages.lrinternal.com/Themes/profile/html/profile.html",
+	})
+	_, err := request.Rest(http.MethodPost, profile, nil, string(bodyProfile))
+	if err != nil {
+		return err
+	}
+
+	var app api.LoginResponse
+	appInfo, err1 := cmdutil.GetCreds()
+	if err1 != nil {
+		return err
+	}
+	err = json.Unmarshal(appInfo, &app)
+	if err != nil {
+		return err
+	}
+
+	appName := app.AppName
```

Comment:
No need for new variable here, we can directly use `app.AppName` in the URL as the type is string